keyboards/massdrop/alt/config_led.h

```diff
@@ -17,6 +17,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
+#ifdef ORGB_ENABLE
+#define ORGB_ZONES_COUNT 1
+#define ORGB_MATRIX_COLUMNS 7
+#define ORGB_MATRIX_ROWS 17
```

Comment:
Should these numbers be around the other way?

Reply:
Not as far as I'm concerned
![image](https://user-images.githubusercontent.com/11999147/99317680-1ee90280-286f-11eb-8785-2cdf5676e5f4.png)

Reply:
I agree the names are backwards.  You have 17 columns and 7 rows in that image.  The functionality is correct, the names are reversed.

Reply:
Fixed